modules/jupyter-widget/package.json

```diff
@@ -25,9 +25,24 @@
   "scripts": {
     "watch": "(cd ../main && npm run build-bundle -- --env.dev) && ln -f ../main/dist/dist.dev.js ./dist/deckgl.dev.js && webpack-dev-server --env.dev --port 8080",
     "build": "webpack",
+    "build:labextension": "npm pack",
     "prepublishOnly": "webpack"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1.10"
+    "@jupyter-widgets/base": "^1.1.10 || ^2",
```

Comment:
This syntax only makes sense when used in `peerDependencies`. This will always install 2.x.